specification/riskiq/Easm/tspconfig.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  "service-dir":
+    default: "sdk/easm"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "examples"
+    output-file: "{azure-resource-provider-folder}/Microsoft.Easm/preview/2023-03-01-preview/easm.json"
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Defender.Easm"
+    namespace: "{package-dir}"
+    clear-output-folder: true
+    model-namespace: false
+  "@azure-tools/typespec-ts":
+    package-dir: "defendereasm-rest"
+    packageDetails:
+      name: "@azure-rest/defender-easm-rest"
+  "@azure-tools/typespec-java":
+    package-dir: "azure-defender-easm"
+    namespace: com.azure.defender.easm
```

Comment:
namespace should be what you have in arch review https://github.com/Azure/azure-sdk/issues/5062 `com.azure.analytics.defender.easm`
https://github.com/Azure/azure-sdk-for-java/pull/36282/files#r1300780400

Reply:
@weidongxu-microsoft I have updated the namspaces using the approved names